tests/unit/test_mysql.py

```diff
@@ -1,24 +1,29 @@
 import os
 import tempfile
 
+from unittest import mock
+
 from . import utils
 
 from hotsos.core.config import setup_config
+from hotsos.core.host_helpers.systemd import SystemdService
+from hotsos.core.issues import IssuesManager
 from hotsos.plugin_extensions.mysql import summary
 from hotsos.core.ycheck.scenarios import YScenarioChecker
 
 
+
 SYSTEMD_UNITS = """
-  jujud-unit-hacluster-mysql-1.service   loaded active     running   juju unit agent for hacluster-mysql/1                                    
-  jujud-unit-mysql-1.service             loaded active     running   juju unit agent for mysql/1                                              
-  mysql.service                          loaded active     running   Percona XtraDB Cluster daemon  
```

Comment:
im guess you have some kind of autopep8 tool doing this? if you could leave as-is that would be better because it mimics the contents of an actual sosreport file (which has the trailing whitespace etc).